docs/src/main/paradox/discovery/aws.md

```diff
@@ -139,10 +139,8 @@ Screenshot of two ECS task instances (the service name is
 
 #### Dependencies and usage (ECS Discovery)
 
-There are two "flavours" of the ECS Discovery module. Functionally they are
-identical; the difference is in which version of the AWS SDK they use. They are
-both provided so that you can choose which set of AWS SDK dependencies you're
-most comfortable with bringing in to your project.
+There are two "flavours" of the ECS Discovery module. One offers service-discovery using
```

Comment:
three now?

Reply:
In this context, there are still only 2 since it refers to blocking vs non-blocking AWS SDK usage.